src/FormControlFeedback.js

```diff
@@ -0,0 +1,64 @@
+import classNames from 'classnames';
+import React from 'react';
+
+import bootstrapUtils, { bsClass } from './utils/bootstrapUtils';
+
+import Glyphicon from './Glyphicon';
+
+const defaultProps = {
+  bsRole: 'feedback',
+};
+
+const contextTypes = {
+  $bs_formGroup: React.PropTypes.object,
+};
+
+class FormControlFeedback extends React.Component {
+  getGlyph(validationState) {
+    switch (validationState) {
+    case 'success': return 'ok';
+    case 'warning': return 'warning-sign';
+    case 'error': return 'remove';
+    default: return null;
```

Comment:
Do you think that's necessary if we already have support for doing a fully custom feedback thing? Like

```js
<FormControl.Feedback>
  <Glyphicon glyph="music" />
</FormControl.Feedback>
```

from the examples?

I feel like I don't want to have two APIs for custom feedback icons, and it seems like it'd be better to only have the more general one.